rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -0,0 +1,104 @@
+:_content-type: REFERENCE
+[id="rosa-release-notes"]
+= {product-title} release notes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-release-notes
+
+toc::[]
+
+{product-title} ROSA is a fully-managed, turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS Site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
+
+[id="about-this-release-section_{context}"]
+== About this release
+
+{product-title} (ROSA) https://access.redhat.com/errata/RHEA-2022:2252[RHEA-2022:2252] is now available. This release uses Kubernetes 1.23 with CRI-O runtime. New features, changes, and known issues that pertain to ROSA 4.10 are included in this topic.
+
+ROSA clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for ROSA, you can deploy OpenShift clusters to either on-premise or cloud environments.
```

Comment:
suggestion to use the correct attribute here for OpenShift rather than "OpenShift clusters" hard-coded, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-name-and-version for info

Reply:
This is done.